pkg/master/master_openapi_test.go

```diff
@@ -0,0 +1,95 @@
+// +build !race
+
+/*
+Copyright 2014 The Kubernetes Authors.
```

Comment:
surprised boilerplate isn't complaining about this... new files are supposed to have 2017